sale_cancel_reason/wizard/cancel_reason_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_sale_order_cancel" model="ir.ui.view">
+        <field name="name">Reason for the cancellation</field>
+        <field name="model">sale.order.cancel</field>
+        <field name="inherit_id" ref="sale.sale_order_cancel_view_form" />
+            <field name="arch" type="xml">
+                <xpath expr="//footer" position="before">
+                    <br />
+                    <p>
+                        Choose the reason for the cancellation of the
+                        sale order.
+                    </p>
+                    <group>
+                        <field name="reason_id" widget="selection" />
```

Comment:
Please remove the selection widget. With many records, filtering becomes complicated, and not all records are displayed.
```suggestion
                        <field name="reason_id" options="{'no_create': True, 'no_open': True}" />
```